tests/unit/hive_metastore/test_migrate.py

```diff
@@ -17,40 +19,81 @@
     errors = {}
     rows = {
         "SELECT": [
             (
                 "hive_metastore",
-                "db1",
-                "managed",
+                "db1_src",
+                "managed_src",
                 "MANAGED",
                 "DELTA",
+                "dbfs:/table_location/table_name",
                 None,
+            ),
+            (
+                "hive_metastore",
+                "db1_src",
+                "managed_src_not_mapped",
+                "MANAGED",
+                "DELTA",
+                "dbfs:/table_location/table_name2",
                 None,
             ),
-        ]
+            (
+                "hive_metastore",
+                "db1_src",
+                "managed_src_db_dataset",
+                "MANAGED",
+                "DELTA",
+                "dbfs:/databricks-datasets",
+                None,
+            ),
+            (
+                "hive_metastore",
+                "db1_src",
+                "managed_src_to_skip",
+                "MANAGED",
+                "DELTA",
+                "dbfs:/table_location/table_name",
+                None,
+            ),
+        ],
+        "skip": [{"key": "databricks.labs.ucx.skip", "value": "true"}],
+        "SHOW TBLPROPERTIES ": [{"key": "fake_key", "value": "fake_value"}],
+        "CREATE TABLE IF NOT EXISTS": [],
     }
     backend = MockBackend(fails_on_first=errors, rows=rows)
     tc = TablesCrawler(backend, "inventory_database")
     client = MagicMock()
-    tm = TablesMigrate(tc, client, backend)
+    tmp = create_autospec(TableMapping)
+    tmp.load.return_value = [
+        Rule("workspace", "ucx_default", "db1_src", "db1_dst", "managed_src", "managed_dst"),
+        Rule("workspace", "ucx_default", "db1_src", "db1_dst", "managed_src_db_dataset", "managed_src_db_dataset"),
+        Rule("workspace", "ucx_default", "db1_src", "db1_dst", "managed_src_to_skip", "managed_src_to_skip"),
+    ]
+    tm = TablesMigrate(tc, client, backend, tmp)
+
     tm.migrate_tables()
 
     assert (list(backend.queries)) == [
         "SELECT * FROM hive_metastore.inventory_database.tables",
-        "CREATE TABLE IF NOT EXISTS ucx_default.db1.managed DEEP CLONE hive_metastore.db1.managed;",
-        "ALTER TABLE hive_metastore.db1.managed SET TBLPROPERTIES ('upgraded_to' = 'ucx_default.db1.managed');",
-        "ALTER TABLE ucx_default.db1.managed SET TBLPROPERTIES ('upgraded_from' = 'hive_metastore.db1.managed');",
+        "SHOW TBLPROPERTIES hive_metastore.db1_src.managed_src",
+        "CREATE TABLE IF NOT EXISTS ucx_default.db1_dst.managed_dst DEEP CLONE hive_metastore.db1_src.managed_src;",
+        "ALTER TABLE hive_metastore.db1_src.managed_src SET TBLPROPERTIES "
+        "('upgraded_to' = 'ucx_default.db1_dst.managed_dst');",
+        "ALTER TABLE ucx_default.db1_dst.managed_dst SET TBLPROPERTIES "
+        "('upgraded_from' = 'hive_metastore.db1_src.managed_src');",
```

Comment:
Please use `databricks.labs.ucx` namespace for table properties (unless the product has claimed this tag and this is for compatibility.